dlt/common/typing.py

```diff
@@ -119,7 +120,34 @@ def asstr(self, verbosity: int = 0) -> str:
         ...
 
 
-def extract_type_if_modifier(t: Type[Any]) -> Type[Any]:
+def get_type_name(t: Type[Any]) -> str:
+    """Returns a human-friendly name of type `t`"""
+    if name := getattr(t, "__name__", None):
```

Comment:
Should we instead use `type(t)` and combine it with `get_origin`?

Reply:
`t` is a type, not instance. what do you mean?

Reply:
Ah I got confused because of `getattr`, it's fine.